pkg/tests/testinfra/testinfra.go

```diff
@@ -536,3 +536,12 @@ func CreateUser(t *testing.T, store db.DB, cfg *setting.Cfg, cmd user.CreateUser
 	require.NoError(t, err)
 	return u
 }
+
+func ContainsFeatureToggle(featureToggles []string, item string) bool {
```

Comment:
this does not seems to be used anywhere. Why is it needed?

Reply:
no! It was used when I thought I had to skip alerting tests, but it turns out we don't 😄